### PR TITLE
Renames for search -> golden_retriever.

### PR DESCRIPTION
**Pull request by daithiocrualaoich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
-[![Build Status](https://travis-ci.org/woofwoofinc/csearch.svg?branch=master)](https://travis-ci.org/woofwoofinc/csearch)
-[![Dependency Status](https://dependencyci.com/github/woofwoofinc/csearch/badge)](https://dependencyci.com/github/woofwoofinc/csearch)
+[![Build Status](https://travis-ci.org/woofwoofinc/golden_retriever.svg?branch=master)](https://travis-ci.org/woofwoofinc/golden_retriever)
+[![Dependency Status](https://dependencyci.com/github/woofwoofinc/golden_retriever/badge)](https://dependencyci.com/github/woofwoofinc/golden_retriever)
 [![Windows Build Status](https://ci.appveyor.com/api/projects/status/nve0g5owu810pi4w/branch/master?svg=true)](https://ci.appveyor.com/project/passy/csearch)
```
Comment (daithiocrualaoich) [inline README.md:3]: @passy AppVeyor URLs not updated in this. Will you arrange this please?

Comment (passy) [inline README.md:3]: I shall.

File: README.md [hunk 2]
```diff
@@ -23,17 +23,17 @@ not already available. Then build and test the project using:
...
 ------
 A [Docker] container definition is provided with installations of the tools
-used to develop CSearch. To use the container, first install Docker if not
-already available and start a Docker terminal. Then create the container by
+used to develop Golden Retriever. To use the container, first install Docker if
+not already available and start a Docker terminal. Then create the container by
 running the following build at the top level of the repository source tree:
 
-    docker build --rm=true -t csearch .
+    docker build --rm=true -t golden_retriever .
```
Comment (daithiocrualaoich) [inline README.md]: What do you think about a short version for `golden_retriever` here? `grrr`?

File: README.md (hunk 1 above)
Comment (passy) [inline README.md:3]: Should be done.

Comment (passy) [inline README.md:3]: But you can never know if something involves Windows.

File: README.md (hunk 2 above)
Comment (passy) [inline README.md]: Genius! :shipit:

**Pull request merged**